modules/core/shared/src/main/scala/gem/math/Declination.scala

```diff
@@ -16,7 +18,7 @@ import gem.syntax.parser._
  * [0 - 90] in terms of the underlying `Angle`. Note that the range is *inclusive* of both poles.
  * @see The helpful [[https://en.wikipedia.org/wiki/Declination Wikipedia]] article.
  */
-sealed abstract case class Declination private (toAngle: Angle) {
+sealed abstract case class Declination protected (toAngle: Angle) {
```

Comment:
Is this privacy downgrade required to make the constructor available in `DeclinationOptics`?  I would have thought that perhaps the `this: Declination.type =>` would give it access even if private.

Reply:
Yes, that's why. I'm kind of surprised it worked actually. In any case it's redundant since the trait is sealed.